portable/CMakeLists.txt

```diff
@@ -1,177 +1,316 @@
 # FreeRTOS internal cmake file. Do not use it in user top-level project
 
 add_library(freertos_kernel_port STATIC
+    # 16-Bit DOS ports for BCC
     $<$<STREQUAL:${FREERTOS_PORT},BCC_16BIT_DOS_FLSH186>:
         BCC/16BitDOS/common/portcomn.c
         BCC/16BitDOS/Flsh186/port.c>
+
     $<$<STREQUAL:${FREERTOS_PORT},BCC_16BIT_DOS_PC>:
         BCC/16BitDOS/common/portcomn.c
         BCC/16BitDOS/PC/port.c>
+
+    # ARMv7-M port for Texas Instruments Code Composer Studio
     $<$<STREQUAL:${FREERTOS_PORT},CCS_ARM_CM3>:
         CCS/ARM_CM3/port.c
         CCS/ARM_CM3/portasm.asm>
+
+    # ARMv7-EM port for Texas Instruments Code Composer Studio
```

Comment:
ARMv7-EM or ARMv7-M?

Reply:
Cortex-M4 and Cortex-M7 are ARMv7E-M rather than just ARMv7-M. Some instructions differ.